### DISCUSSION TITLE
部署到服务器上后启动gocqhttp会去github下载模块，但由于没有vpn无法科学上网这一步就报错了

### DISCUSSION THREAD

**Discussion by icewizardry** [nonebot/discussions#252] [问答] [answered]
启动后它自动去下载这个包，但是报错了，因为网路问题请求不到，该如何离线更新呢？
![image](https://user-images.githubusercontent.com/5187301/203014397-aca945ee-7d75-4f4c-af1a-90c8dfb6c2f9.png)

**Comment by yanyongyu** [comment, marked as answer]
@mnixry

**Reply by mnixry** [reply]
修了
